python-frontend/src/main/java/org/sonar/python/types/v2/ClassType.java

```diff
@@ -210,6 +209,10 @@ public String name() {
     return name;
   }
 
+  public String fullyQualifiedName() {
+    return fullyQualifiedName;
+  }
```

Comment:
_nit_ What about adding `@Nonnull` here?
```suggestion
  @Nonnull 
  public String fullyQualifiedName() {
    return fullyQualifiedName;
  }
```

Reply:
makes sense, thanks!